CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.12)
 project(appfwk VERSION 1.1.0)
 
-set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../../daq-buildtools/cmake ${CMAKE_MODULE_PATH})
+set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../daq-buildtools/cmake ${CMAKE_MODULE_PATH})
```

Comment:
Actually, these lines should be removed entirely.

The "true" path is wherever `daq-buildtools` is installed.  The standard way for CMake to find packages is via `CMAKE_PREFIX_PATH`.

Reply:
Thanks.